app/src/main/res/layout/alpha_picker_button.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
File should be named `view_button_alpha_picker` because it's used in a (custom-) view class and is a button.